build/Dockerfile

```diff
@@ -18,6 +18,9 @@ RUN apk add --no-cache \
   netcat-openbsd \
   xmlsec-dev \
   tzdata \
+  wv \
+  poppler-utils \
+  tidyhtml \
```

Comment:
why we are adding those things? this is needed to run mattermost?

Reply:
This are dependencies for the document extraction that is now part of mattermost document search.